### ISSUE TITLE
Import and Export Filters as CSV

### ISSUE THREAD

**Issue by mienaiKnife** [mastodon/mastodon#30174]
### Pitch

In Settings > Data Export, add a row for filters and allow users to export a CSV with their filters to be imported into a different account.

### Motivation

I'm a big fan of how Mastodon handles keyword filtering and have used the feature extensively, and I would like it to be easier to transfer 40+ keyword filters from one of my accounts to another without necessarily migrating accounts or manually copypasting filtered keywords.

**Comment by llutz** [comment]
That feature would be very usefull, also nice to share filterlists.

**Comment by LVCTVS** [comment]
Yes, please! This and also extending the filter functions. Maybe simple regex and more fields for the context!

**Comment by markwyner** [comment]
It's looks like everywhere this feature was suggested and added as a suggestion, @renchap has gone through and removed all of them. I wonder why? Is this because it's being ignored, or because they're working on it?

**Comment by renchap** [comment]
We switched to the new Github Issue types rather than labels, so all `suggestion` labels have been removed from issues and replaced by the `Suggestion` type.

![Image](https://github.com/user-attachments/assets/c21095ce-4f6a-4bfa-99c1-cb49e98f49d6)

**Comment by markwyner** [comment]
Thanks for the clarification, @renchap.

**Comment by Cassolotl** [comment]
@LVCTVS 

> also extending the filter functions. Maybe simple regex and more fields for the context!

I think this should probably be a new issue, rather than appended to this one?

**Comment by renchap** [comment]
Duplicate of #9177
